### DISCUSSION TITLE
Any way to access disk of broken VM?

### DISCUSSION THREAD

**Discussion by thstraus** [lima-vm/lima#2200] [Q&A] [answered]
Hi everybody,

I have a project development environment installed on a Lima VM of type "vz" that has been broken and is unresponsive at the step waiting for ssh to come up:

> INFO[0601] [hostagent] Waiting for the essential requirement 1 of 3: "ssh" 
> INFO[0604] [hostagent] 2024/02/09 15:35:25 tcpproxy: for incoming conn 127.0.0.1:55080, error dialing "192.168.5.15:22": connect tcp 192.168.5.15:22: no route to host 

This is not the first time it happens to me, and I don't think I'm the only one. I just started a new VM from scratch. This time I'm having a hard time to get my environment up and it could be helpful if I only could access the disk of the broken VM.

Not sure this is possible, but anyhow I'm asking, in case there are some ideas how I could access the filesystem. Any way to mount it? Any hint is really welcome!

Thanks,

Thomas

**Comment by afbjorklund** [comment, marked as answer]
* https://github.com/lima-vm/lima/discussions/2104
* https://github.com/lima-vm/lima/issues/2115

**Reply by thstraus** [reply]
So, I'm not the first asking. Sorry for that and many thanks for the references!

**Reply by afbjorklund** [reply]
It needs to make its way into the documentation, but should give you some hints to start with
